### ISSUE TITLE
Words with no lemmas

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Uyghur-UDT#1]
Quite a few words in the treebank have no lemmas on the words, such as

```
# sent_id = s1802
# text = سەن شۇ چاققىچە سەۋر قىلىپ تۇرالامسەن؟
1	سەن	سەن	PRON	P	Case=Nom|Number=Sing|Person=2|PronType=Prs	6	nsubj	_	Translit=sen
2	شۇ	شۇ	PRON	P	Case=Nom|PronType=Dem	3	det	_	Translit=shu
3	چاققىچە	_	NOUN	N	_	6	obl	_	Translit=chaqqiche|Morf=Unknown
4	سەۋر	سەۋر	NOUN	N	Case=Nom	5	compound	_	Translit=sewr
5	قىلىپ	قىل	VERB	V	Aspect=Perf|VerbForm=Conv	6	advcl	_	Translit=qilip
6	تۇرالامسەن	_	VERB	V	_	0	root	_	SpaceAfter=No|Translit=turalamsen|Morf=Unknown
7	؟	؟	PUNCT	Y	_	6	punct	_	Translit=?
```

**Comment by dan-zeman** [comment]
Yes. The lemmatizer that was used has limited coverage, but some lemmas are IMHO better than no lemmas.

Leaving this issue open as a reminder that there is room for improvement – although there is no promise that anything will improve soon.

**Comment by XIRALIP** [comment]
What environment is required? Are there any examples that I can run locally?

**Comment by dan-zeman** [comment]
It depends on what you mean by "running the examples". The CoNLL-U file format is a text format, so the simplest (though not necessarily the most convenient) way is to open it in in a text editor and edit it there.

**Comment by XIRALIP** [comment]
There are sentences and words in the dataset. Can we use this framework to automatically mark it to CoNLL-U  file?
I want to mark my dataset for LLM.

**Comment by dan-zeman** [comment]
There are trainable parsers (such as UDPipe or Stanza) that can be trained on this dataset, if that's what you are asking about.
